ssl/record/ssl3_record.c

```diff
@@ -663,6 +663,15 @@ int ssl3_get_record(SSL *s)
             goto f_err;
         }
 
+        /* If client see a packet overflowing his requested MFL setting */
```

Comment:
Bad assumption from my debug sessions ...
Without it, when max_fragment_size is lower than active MFL limit , I get a :
`# 5136:error:14068044:SSL routines:do_ssl3_write:internal error:ssl\record\rec_layer_s3.c:830:`

Last test statement of an if clause is falling :
```
...
                    && !WPACKET_reserve_bytes(thispkt, maxcomplen,
                                              &compressdata))) {
            SSLerr(SSL_F_DO_SSL3_WRITE, ERR_R_INTERNAL_ERROR);

```